tests/e2e/gov/tx.go

```diff
@@ -545,7 +545,8 @@ func (s *E2ETestSuite) TestNewCmdWeightedVote() {
 	for _, tc := range testCases {
 		tc := tc
 		s.Run(tc.name, func() {
-			cmd := cli.NewCmdWeightedVote()
+			// TODO HV2: changed from NewCmdWeightedVote to NewCmdVote. Fix tests accordingly.
+			cmd := cli.NewCmdVote()
```

Comment:
Even if weighted voting is not supported in Heimdall, would it nevertheless be simpler to leave this as `NewCmdWeightedVote` so that we spend less time re-doing the tests? (at the end of the day, this is test code)
Or are you planning on testing something else than what is tested in vanilla SDK `v0.50.x`?

This would also apply to file `x/feegrant/client/cli/tx_test.go` below.

Reply:
Right, I agree. Changed it in the very beginning with the purpose of implementing some tests, but both are actually covered in the vanilla SDK. Reverted [here](https://github.com/0xPolygon/cosmos-sdk/pull/15)

Reply:
Resolved